package.json

```diff
@@ -5,16 +5,9 @@
   "main": "main.js",
   "scripts": {
     "start": "electron .",
-    "dev": "electron . --enable-logging",
     "clean": "rm -rf ./dist",
```

Comment:
this is not cross platform, if you want windows support you can use a pckage called `rimraf` to do that. here is the [link](https://www.npmjs.com/package/rimraf)